parcels/collection/collections.py

```diff
@@ -789,8 +786,8 @@ class ParticleCollection(Collection):
     _data = None  # formerly: particle_data
 
     def __init__(self):
-        """
-        ParticleCollection - Constructor
+        """Constructor for ParticleCollection.
+
```

Comment:
Not sure about this. The capitalisation of the class name was making D403 act up, but this rewording is now inconsistent with other parts. Same for the edit just below

Reply:
Hmm, I don't understand the problem here. This passes `flake8` now, right? But you would want to revert it? Seems fine to me

Reply:
Passes with flake8, just now its inconsistent with:
https://github.com/VeckoTheGecko/parcels/blob/0ee5bd2f82cb25650560af9b59eceb575d147620/parcels/collection/collections.py#L22

https://github.com/VeckoTheGecko/parcels/blob/0ee5bd2f82cb25650560af9b59eceb575d147620/parcels/collection/collections.py#L31

https://github.com/VeckoTheGecko/parcels/blob/0ee5bd2f82cb25650560af9b59eceb575d147620/parcels/collection/collectionaos.py#L151

https://github.com/VeckoTheGecko/parcels/blob/0ee5bd2f82cb25650560af9b59eceb575d147620/parcels/collection/collectionsoa.py#L165

Reply:
OK I see. I think that at some point we'll have to carefully go through the docstrings again anyway, and then we can homogenise this inconsistency. It doesn't bother me that much.

Reply:
Easy, that works. It was a minor point anyway :)